docs/src/app/components/AppBar/ExampleIconMenu.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import AppBar from 'material-ui/app-bar';
```

Comment:
try running `npm run build -- --watch` on the root of the project to see if it helps.